Brokerages/Brokerage.cs

```diff
@@ -100,6 +100,11 @@ protected Brokerage(string name)
         public abstract void Disconnect();
 
         /// <summary>
+        /// Dispose of the brokerage instance
+        /// </summary>
+        public abstract void Dispose();
```

Comment:
Perhaps we could make this `virtual` instead (with the empty implementation), so we only need the override in IB brokerage ?